carstore/README.md

```diff
@@ -0,0 +1,41 @@
+# Carstore
+
+Store a zillion users of PDS-like repo, with more limited operations (mainly: firehose in, firehose out).
+
+## [ScyllaStore](scylla.go)
+
+Blocks stored in ScyllaDB.
+User and PDS metadata stored in gorm (PostgreSQL or sqlite3).
+
+## [FileCarStore](bs.go)
+
+Store  'car slices' from PDS source subscribeRepo firehose streams to filesystem.
+Store metadata to gorm postgresql (or sqlite3).
+Periodic compaction of car slices into fewer larger car slices.
+User and PDS metadata stored in gorm (PostgreSQL or sqlite3).
+FileCarStore was the first production carstore and used through at least 2024-11.
+
+## [SQLiteStore](sqlite_store.go)
+
+Experimental/demo.
+Blocks stored in trivial local sqlite3 schema.
+Minimal reference implementation from which fancy scalable/performant implementations may be derived.
+
+```sql
+CREATE TABLE IF NOT EXISTS blocks (uid int, cid blob, rev varchar, root blob, block blob, PRIMARY KEY(uid,cid))
```

Comment:
This will store all blocks for a user in a single partition FYI, you'd want `PRIMARY KEY((uid, cid)) for each block to be its own partition.